deps/src/coeffs.cpp

```diff
@@ -44,9 +44,8 @@ void singular_define_coeffs(jlcxx::Module & Singular)
         }
     });
 
-    Singular.method("n_Delete_Q", [](void * n, coeffs cf) {
-        number   tt = reinterpret_cast<number>(n);
-        number * t = &tt;
+    Singular.method("n_Delete_Q", [](snumber * n, coeffs cf) {;
```

Comment:
```suggestion
    Singular.method("n_Delete_Q", [](snumber * n, coeffs cf) {
```
BTW, all places using `snumber *` could also just use `number`

Reply:
I would say it works, otherwise the tests would segfault.